package.json

```diff
@@ -11,7 +11,8 @@
     "change": "beachball change",
     "clean": "lerna run clean --stream -- --color",
     "format": "node packages/scripts/formatFiles.js -i -style=file -assume-filename=../.clang-format",
-    "format:verify": "node packages/scripts/formatFiles.js -i -style=file -assume-filename=../.clang-format -verify"
+    "format:verify": "node packages/scripts/formatFiles.js -i -style=file -assume-filename=../.clang-format -verify",
+    "test": "lerna run test --stream -- --ci"
```

Comment:
I have mixed feelings about running these in parallel. This will break if people add more E2E style tests that require focus, and Jest will do some level of parralelization already IIRC, but it seems conceptually better to parralelize still. Seems like we can change if we run into issues.